src/plugins/demo/README.md

```diff
@@ -0,0 +1,10 @@
+# The Demo Plugin
+
+This plugin exists for testing convenience. It's nice sometimes to have a super
```

Comment:
Is this plugin also supposed to be a structural model for real plugins?

If not, can we call that out explicitly?

Reply:
Great point. Updated README.